apps/docs/reference/cli.md

```
@@ -193,6 +194,39 @@ one:
+```bash
+substrat push --check           # layer rules + the declared permission surface
+substrat push --check --json    # the same surface as data, for a CI diff
+```
+
+Runs everything a push does **locally** and stops there: the layer rules, then your declared
+permission surface — resolve `"substrat": { "permissions" }`, import the entry, derive the
+registry, print every key with the module that declares it and its description, every role,
+every entity-grant shape, and the digest promotion compares. It needs **no login and makes no
+network call**, so it belongs in a pull-request job:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)